packages/core/src/components/context-menu/contextMenuTarget.tsx

```diff
@@ -12,10 +12,11 @@ import * as ContextMenu from "./contextMenu";
 
 export interface IContextMenuTarget extends React.Component<any, any> {
     renderContextMenu(e: React.MouseEvent<HTMLElement>): JSX.Element ;
+    onCloseContextMenu?(): void;
```

Comment:
Our other component APIs typically put verbs at the end of long callback names (e.g. I've recently seen `onDayMouseEnter`, `onDayMouseLeave`, `onDayClick`, `onValueChange`).